### DISCUSSION TITLE
I would love to help develop an iOS app for this

### DISCUSSION THREAD

**Discussion by hainayanda** [filebrowser/filebrowser#2612] [Ideas]
I'm using Filebrowser to easily share my home server directory with my household. But since my whole household are using iOS devices, it will be nice if we have an iOS app to browse the file instead of using a web browser, especially when we want to share and back up photos and videos.
I think in my free time I could help develop an iOS app for Filebrowser so managing files is easy from iPhone or iPad. Not just for my household but for communities too.
What do you think?

**Comment by marketingBBB** [comment]
https://amzn.to/44WQGlX

**Comment by underthebanyan** [comment]
I agree. A companion app would be pretty stellar, but I don't think photos and videos are the right thing to focus on. Immich is an excellent self-hosted alternative to google photos and is an incredible application for that particular purpose. I've been using it for my family and it's been pretty amazing. Highly recommend you look into it. 

Regardless, definitely in support of building an ios app for filebrowser.
